src/Components/Books.js

```diff
@@ -12,19 +13,51 @@ const Books = (props) => {
   };
 
   return (
-    <div className="d-flex flex-column align-items-center">
-      <div className="border border-secondary rounded  w-75 my-3 ps-5">
-        <p className="text-left">{type}</p>
-        <p>{title}</p>
-        <p>{author}</p>
-
-        <button
-          type="button"
-          className="btn btn-primary"
-          onClick={clickHandler}
-        >
-          Remove
-        </button>
+    <div>
+      <div className="book-card">
+        <div>
+          <p className="category">{type}</p>
+          <p className="Title">{title}</p>
+          <p className="author">{author}</p>
+          <div className="action-buttons">
+            <span type="button" className="Remove">
+              Comments
+            </span>
+            <div className="Line-2" />
+            <span type="button" onClick={clickHandler} className="Remove">
+              Remove
+            </span>
+            <div className="Line-2" />
+            <span type="button" className="Remove">
+              Edit
+            </span>
+          </div>
+        </div>
+        {/* progress bar try */}
+        <div className="middle-section">
+          <div className="wrap">
+            <div className="circle">
+              <div className="mask full">
+                <div className="fill" />
```

Comment:
[REQURED] Kindly fix the errors mentioned in the image below.

![image](https://user-images.githubusercontent.com/62938735/194722415-84f6072d-59df-45ce-9d0f-dbffeba74745.png)